test/metabase/api/dashboard_test.clj

```diff
@@ -3097,10 +3097,9 @@
 (deftest valid-filter-fields-test
   (testing "GET /api/dashboard/params/valid-filter-fields"
     (letfn [(result= [expected {:keys [filtered filtering]}]
-                (testing (format "\nGET dashboard/params/valid-filter-fields")
```

Comment:
This just added a useless line really, no new information. I guess it's a leftover from some test refactoring.